src/agentscope/pipelines/pipeline.py

```diff
@@ -72,6 +76,7 @@ def __init__(
         self.condition_func = condition_func
         self.if_body_operator = if_body_operators
         self.else_body_operator = else_body_operators
+        self.participants = [self.if_body_operator] + [self.else_body_operator]
```

Comment:
The type `Operators` maybe a sequence of `Operator`, please ensure the operation in line 79 is correct. 
`Union[Operator, Sequence[Operator]]`